apps/mobile/src/components/Feed/Posts/PostListItem.tsx

```diff
@@ -55,6 +66,15 @@ export function PostListItem({ feedPostRef, queryRef }: Props) {
   });
 
   const firstToken = feedPost.tokens?.[0] || null;
+  const community = firstToken?.community ?? null;
+
+  const handleCommunityPress = useCallback(() => {
+    if (!community || !community.contractAddress) return;
+    navigation.push('Community', {
+      contractAddress: community.contractAddress?.address ?? '',
+      chain: community.contractAddress?.chain ?? '',
+    });
```

Comment:
nit: should we also check for the existence of `.address` and `.chain` in the `if` statement and return early if they don't exist? doesn't make sense to push to a community page with empty string params right?

Reply:
good call! inverted the condition so we only push if address and chain are present